andela/ah-frontend-taces
# #159951818 Add Password reset feature

### What does this PR do?
- Adds the password reset feature to Author's Haven Frontend

### Description of Task to be completed?
- This feature makes a user able to reset their passwords using a password reset link sent to their emails

### How should this be manually tested?
-  Clone the app.
- Navigate to the login feature and click reset your password
- Type in a registered user's email and submit
- click the reset link sent and fill in the forms visible and submit
- You should be presented with a login modal, then you can log in again

### Any background context you want to provide?
- I created a password reset component
- Added it to the login feature and to the Homepage

### What are the relevant pivotal tracker stories?
- [#159951818](https://www.pivotaltracker.com/story/show/159951818)
